enterprise/frontend/src/embedding-sdk/components/private/SdkUsageProblem/SdkUsageProblemBanner.tsx

```diff
@@ -0,0 +1,144 @@
+import { useDisclosure } from "@mantine/hooks";
+import cx from "classnames";
+import { useState } from "react";
+
+import { SDK_SSO_DOCS_LINK } from "embedding-sdk/lib/usage-problem";
+import type { SdkUsageProblem } from "embedding-sdk/types/usage-problem";
+import LogoIcon from "metabase/components/LogoIcon";
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { originalColors } from "metabase/lib/colors";
+import {
+  Button,
+  Card,
+  Flex,
+  Icon,
+  Popover,
+  Stack,
+  Text,
+  useMantineTheme,
+} from "metabase/ui";
+
+import S from "./SdkUsageProblemBanner.module.css";
+
+interface Props {
+  problem: SdkUsageProblem | null;
+}
+
+// Prevent the usage problem banner from inheriting the theme colors,
+// so they remain legible even when the theme is changed.
+const unthemedBrand = originalColors["brand"];
+const unthemedTextDark = originalColors["text-dark"];
+const unthemedTextMedium = originalColors["text-medium"];
+
+export const SdkUsageProblemBanner = ({ problem }: Props) => {
+  const theme = useMantineTheme();
+
+  const [expanded, setExpanded] = useState(false);
+  const [shown, { close: hideBanner }] = useDisclosure(true);
+
+  if (!problem || !shown) {
+    return null;
+  }
+
+  const { severity } = problem;
+  const isError = severity === "error";
+
+  // When the font family cannot be loaded from the MB instance,
+  // due to MB instance outage or missing CORS header,
+  // we fall back to the system font.
+  const fontFamily = `${theme.fontFamily}, sans-serif`;
+
+  return (
+    <Popover position="top-start" opened={expanded} onChange={setExpanded}>
```

Comment:
is "top-start" for the text direction?
Do we show the visual clue on the right when the language direction is right-to-left?

Reply:
@npretto this is for the position of the popover itself, so it would place the popover at trigger's (flex)-start on the top of the trigger. The visual cue should still be on the left.

Reply:
I was wondering if it would display in the wrong place on RTL languages, but I tested and it's ok